src/views/Application.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div class="container py-4">
+    <div class="row">
+      <div class="col">
+        <a-card hoverable style="width: 100%">
+          <template class="ant-card-actions" slot="actions">
+            <a-icon type="api" title="Instalar agora" @click="installApp" v-if="!installed"/>
+            <a-icon type="setting" title="Configurar aplicativo" @click="tongleSettings" v-if="installed"/>
+            <a-popconfirm
+              title="Tem certeza que deseja excluir o aplicativo?"
+              @confirm="deleteApp"
+              okText="Sim"
+              cancelText="Não"
+              v-if="installed"
+            >
+              <a-icon type="delete" title="Excluir aplicativo"/>
+            </a-popconfirm>
+          </template>
+          <a-card-meta :title="title" :description="shortDescription">
+            <a-avatar
+              slot="avatar"
+              :src="icon"
+            />
+          </a-card-meta>
+        </a-card>
+      </div>
+    </div>
+    <div class="row pt-4" v-if="showSettings">
+      <div class="col">
+        <a-card hoverable style="width: 100%">
+          <a-collapse :bordered="false" v-for="(setting, index) in application.json_body" :key="index">
```

Comment:
Campos de configuração deveria vir do `application.admin_settings` da Store API, não do market.
É necessário que ele possa ver e configurar aplicativos que possivelmente não estão listados no market.

Reply:
Eu pensei em usar o admin_settings vindo do market porque assim, se o aplicativo for atualizado no market, o usuário não precisa instalar o aplicativo novamente pra usar as novas configurações.

Reply:
Mas aí você limita o usuário, não será possível customizar os campos via API, nem usar apps não listados no market.

Além disto, possivelmente o admin settings influencia em scripts no frontend, mudar a versão só do admin settings pode resultar em `data` em uma versão não correspondente ao script carregado, e isto deve resultar em erros fatais, entendeu?
Tudo precisa estar atualizado (ou em uma versão estável), por isto os apps são versionáveis.

A priori, se o app for atualizado o merchant deverá atualizar na sua loja também.

Reply:
ok